### ISSUE TITLE
pytorch2caffe failed at add operation

### ISSUE THREAD

**Issue by andyhahaha** [xxradon/PytorchToCaffe#10]
I'm trying convert RefineDet to caffe model. RefineDet only contain regular CNN module(conv2d, convtranspose2d, maxpool, batchnorm).
https://github.com/luuuyi/RefineDet.PyTorch
However, pytorch2caffe failed at add operation as follow:
![image](https://user-images.githubusercontent.com/14272969/54814090-f7244e80-4cc9-11e9-978e-034cf03bfbbb.png)

This add operation is from here
https://github.com/luuuyi/RefineDet.PyTorch/blob/0e4b24ce07245fcb8c48292326a731729cc5746a/layers/modules/l2norm.py#L20
Why pytorch2caffe failed at a simple operation?
Thanks for help!

**Comment by bigbrother33** [comment]
I met same problem,did you solve it ?

**Comment by ernestlishun** [comment]
met same problem, can you tell how to solve it, thank you
